cas-server-core-tickets/src/main/java/org/jasig/cas/ticket/registry/DefaultTicketRegistry.java

```diff
@@ -47,7 +47,7 @@
  * @since 3.0.0
  */
 @Component("defaultTicketRegistry")
-public final class DefaultTicketRegistry extends AbstractTicketRegistry implements Job {
+public final class DefaultTicketRegistry extends AbstractCrypticTicketRegistry implements Job {
```

Comment:
If everything is moving under `AbstractCrypticTicketRegistry`, shouldn't `AbstractCrypticTicketRegistry` be merged into `AbstractTicketRegistry`?

Again, different patch...